src/fetchQueue.js

```diff
@@ -1,235 +1,235 @@
-module.exports = (function () {
-  'use strict';
-
-  var $ = require('jquery');
+var $ = require('jquery');
+
+/**
+ * This class implements a queue for Deferred objects.  Whenever one of the
+ * objects in the queue completes (resolved or rejected), another item in the
+ * queue is processed.  The number of concurrently processing items can be
+ * adjusted.  At this time (2015-12-29) most major browsers support 6
+ * concurrent requests from any given server, so, when using the queue for
+ * tile images, thie number of concurrent requests should be 6 * (number of
+ * subdomains serving tiles).
+ *
+ * @class
+ * @alias geo.fetchQueue
+ *
+ * @param {object} [options] A configuration object for the queue.
+ * @param {number} [options.size=6] The maximum number of concurrent deferred
+ *    objects.
+ * @param {number} [options.track=600] The number of objects that are tracked
+ *    that trigger checking if any of them have been abandoned.  The fetch
+ *    queue can grow to the greater of this size and the number of items that
+ *    are still needed.  Setting this to a low number will increase
```

Comment:
I think this sentence could be improved further .

Reply:
Suggestions welcome.  Though this wasn't altered by this PR, so such changes could be a different PR.